### ISSUE TITLE
1.12: Applied Energistics Textures not applying

### ISSUE THREAD

**Issue by IronPiston** [CyanideX/Unity#166]
I am using AE2 rv5 stable 11 and the latest version of Unity for 1.12, and textures are not applying. Other textures are applying, so I assume the issue lies with the mod changing the naming scheme of textures.

**Comment by CyanideX** [comment]
Yeah, it's the naming scheme. They've finally named the files properly and have gotten rid of the 'block' prefix on each texture. It's going to take some time to fix and I don't think anyone is available at the moment. You're more than welcome to make the changes and submit a PR. 😉 

Or maybe we can call on the great @SnowShock35. ❤️

**Comment by IronPiston** [comment]
I'd be willing to fix it, but I have next to no experience using git...
Once I made it, how would I go about doing a pull request?

**Comment by CyanideX** [comment]
If you're unsure, you can always link to a download of a zip file containing the files. 😃

**Comment by JelmarG** [comment]
I have actually been 'working' on and off renaming all AE2 textures for Unity. I can provide a zip at some point. Probably a few days, because it's boring work and I do a bit every now and then. I think some blocks might have changed a little bit as well, like 'quartz_fixture_metal' and 'quantum_ring_light', I didn't see those and some others in the original Unity textures, but at least the zip will contain renamed images that do exist. I'm sure you can put on the finishing touches if anything is missing or incorrect, @CyanideX ?

**Comment by CyanideX** [comment]
That would be great, @JelmarG! I haven't updated the mod textures in a while so I'm sure there are a few things missing.

**Comment by Jwigou** [comment]
@JelmarG use "bulk rename utility", it should get the work done in no time, I'm just too busy at the moment and couldn't get around to doing so.

**Comment by CyanideX** [comment]
@Jwigou, a lot of texture names have been changed entirely. This isn't something we can do in bulk unfortunately.

**Comment by JelmarG** [comment]
Yeah, no, this can't be bulk renamed, unfortunately. I have to carefully check the AE2 directories to see what goes where, and then manually rename the textures. I'm quite far along, though, but as I said, I fiddle with it in between work and other stuff, so it takes a while.

**Comment by JelmarG** [comment]
We should probably wait for this: [Texture/Model cleanup](https://github.com/AppliedEnergistics/Applied-Energistics-2/issues/3546)

**Comment by Fortist** [comment]
Did this over the past hour or so.
https://mega.nz/#!aIEHwQoC!3gWn-CvS4GXZLYgQL2RlLbhnJa6LTJvHNYmhy9LE-iY
A lot of the textures were already the same as the default textures, so I removed them as best I can, but if I deleted some changed ones, someone else can go over it.

Im gonna test and see if these work, but they should. I can reupload onto a different service if mega isnt your cup of tea.

**Comment by fscan** [comment]
FYI, the cleanup is planned for the 1.13 update together with all the necessary naming changes.  This is probably some time away .. depends when MC releases and how fast forge updates.

**Comment by ruifung** [comment]
@Fortist You have a naming issue in that file, for charged certus crystals, you have _png.mcmeta instead of .png.mcmeta
